shared/src/Datadog.Trace.ClrProfiler.Native/single_step_guard_rails.cpp

```diff
@@ -280,7 +300,10 @@ void SingleStepGuardRails::SendTelemetry(const std::string& runtimeName, const s
         + "\",\"language_name\": \"dotnet\",\"language_version\": \"" + runtimeVersion
         + "\",\"tracer_version\": \"" + PROFILER_VERSION
         + "\",\"pid\":" + std::to_string(GetPID())
-        + "},\"points\": " + points + "}";
+        + ",\"inject_result\": \"" + m_injectResult
+        + "\",\"inject_result_reason\": \"" + m_injectResultReason
+        + "\",\"inject_result_class\": \"" + m_injectResultClass
```

Comment:
Given that there's no guarantee `SetInjectResult` has been called, should we check if the values have been set, and if not give them a default value? 🤔

Reply:
Oh yes, you're right, that's fine, thanks!